src/js/script.js

```diff
@@ -0,0 +1,78 @@
+
+let inputBtn = document.querySelector('.input__btn'),
+ inputUser =  document.querySelector('.input__user'),
+ toDo = document.querySelector('.maine');
```

Comment:
I have a couple of comments:
1. Pay attention to the spelling
2. It is not the best way to declare the variable in one line. It is better to separate them, especially when you initialise them
3. As I see, you're not going to redeclare them, so you should use const
4. If these elements should be exclusively unique, you can use id instead of class
5. It is very important to name the variable in such way you understand its meaning without looking at other files or additional context.
```suggestion
const plusButton = document.getElementById('plus_button');
const taskInput =  document.getElementById('task_input');
const taskList = document.getElementById('main');
```

Reply:
Hi, thanks for your comments, fixed this in my code.